src/types/drupal/node.ts

```diff
@@ -158,6 +165,32 @@ export interface NodeHealthCareLocalFacility extends DrupalNode {
   field_operating_status_facility: string
   field_region_page: NodeHealthCareRegionPage
 }
+export interface NodeVetCenter extends DrupalNode {
+  field_address: FieldAddress
+  field_cc_non_traditional_hours: FieldCCText
+  field_cc_vet_center_call_center: FieldCCText
+  //TODO Fix after faq work
```

Comment:
are these TODOs still valid after the other accordion PRs? Do we know the types now?

Reply:
Yes. added those types and refactored the prepareForVisit section now.